bigchaindb/commands/bigchaindb.py

```diff
@@ -278,6 +292,16 @@ def create_parser():
                                help='The backend to use. It can be either '
                                     'rethinkdb or mongodb.')
 
+    validator_parser = subparsers.add_parser('upsert-validator',
+                                             help='Add/update/delete a validator')
+
+    validator_parser.add_argument('public_key',
+                                  help='Public key of the new validator.')
```

Comment:
It might not be a _new_ validator. It might be an existing validator that's being updated or deleted. Maybe `'Public key of the validator.'`?